moira/api/resources/notification.py

```diff
@@ -14,12 +14,15 @@ def __init__(self, db):
     def render_GET(self, request):
         notifications, total = yield self.db.getNotifications(request.args.get('start')[0],
                                                               request.args.get('end')[0])
-        self.write_json(request, {"list": notifications, "total": total})
+        result = []
+        for jsonstr in notifications:
+            result.append(jsonstr)
```

Comment:
Perhaps there was tricky bug with generators. But i do not remember unfortunately.